package-lock.json

```diff
@@ -31575,7 +31575,8 @@
       "dependencies": {
         "@mongosh/async-rewriter2": "0.0.0-dev.0",
         "@mongosh/history": "0.0.0-dev.0",
-        "@mongosh/shell-api": "0.0.0-dev.0"
+        "@mongosh/shell-api": "0.0.0-dev.0",
+        "@mongosh/types": "0.0.0-dev.0"
```

Comment:
I think this is leftover from when `mongosh/types` was a prod dependency of shell-evaluator and should be removed